src/App.tsx

```diff
@@ -1,7 +1,32 @@
-import React from 'react';
+import { Component } from 'react';
+import './index.scss';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.updatePressedKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.updatePressedKey);
+  }
+
+  updatePressedKey = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: `The last pressed key is [${event.key}]` });
```

Comment:
String `The last pressed key is [${event.key}]` is not a key type
```suggestion
    this.setState({ pressedKey: event.key });
```